docs/source/usage/reference.rst

```diff
@@ -5,7 +5,7 @@ Reference
 
 .. sectionauthor:: Axel Huebl
 
-PIConGPU is a year-long, scientific project with many people contributing to it.
+PIConGPU is an almost decade-long scientific project with many people contributing to it.
```

Comment:
Note: previously used `year-long` is wrong here, since it means 1 year. In case you don't like my version, at least change to `multiyear`.

Reply:
Like it, well done thx.